### PR TITLE
chore(package): cross env

### PR DESCRIPTION
**Pull request by KristijanJovanovski**
Added cross env package.
Also window as any type due to ts lint and compiler errors.

### CODE REVIEW

File: src/client/store/configureStore.ts [hunk 1]
```diff
@@ -12,8 +12,8 @@ const createEnhancer = () => {
   const composeEnhancers =
     process.env.NODE_ENV !== 'production' &&
     typeof window === 'object' &&
-    window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
-      ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({})
+    (window as any).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
```
Comment (hiroppy) [inline src/client/store/configureStore.ts]: I think this is necessary. see https://github.com/hiroppy/ssr-sample/blob/master/src/types/global.d.ts

**Comment by hiroppy** [comment]
@KristijanJovanovski thank you for submitting PR! I left a comment:)

File: src/client/store/configureStore.ts (hunk 1 above)
Comment (KristijanJovanovski) [reply src/client/store/configureStore.ts]: Yeah, you are right. I'll revert changes made to ts files.
cross-env should be merged though

**Comment by hiroppy** [comment]
Thanks!

**Pull request merged**